int/tests/test_connections.py

```diff
@@ -1,4 +1,5 @@
 """Connections Tests"""
+from re import M
```

Comment:
This seems like an odd addition.. Auto-import getting trigger happy or something?